src/flows/recommendation_api/corpus_candidate_sets/recommended_by_recency.py

```diff
@@ -16,45 +16,55 @@
 
 
 EXPORT_RECOMMENDED_CANDIDATE_SET_SQL = """
-WITH prep as (
+WITH recently_updated_items as (
     SELECT 
         approved_corpus_item_external_id as "ID", 
         topic as "TOPIC",
         reviewed_corpus_item_updated_at as "REVIEW_TIME" 
     FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS" 
-    WHERE REVIEWED_CORPUS_ITEM_UPDATED_AT >= DATEADD('day', %(MAX_AGE_DAYS)s, current_timestamp())
-    AND CORPUS_REVIEW_STATUS = 'recommendation'
+    WHERE CORPUS_REVIEW_STATUS = 'recommendation'
     AND SCHEDULED_SURFACE_ID = 'NEW_TAB_EN_US'
     AND NOT is_syndicated
     AND NOT is_collection
-    
-    UNION
-    
+),
+
+recently_scheduled_items as (    
     SELECT 
         approved_corpus_item_external_id as "ID", 
         topic as "TOPIC",
         scheduled_corpus_item_scheduled_at as "REVIEW_TIME"
     FROM "ANALYTICS"."DBT"."SCHEDULED_CORPUS_ITEMS"
-    WHERE SCHEDULED_CORPUS_ITEM_SCHEDULED_AT BETWEEN DATEADD('day', %(MAX_AGE_DAYS)s, current_timestamp()) AND current_timestamp()
+    WHERE SCHEDULED_CORPUS_ITEM_SCHEDULED_AT < current_timestamp()
     AND CORPUS_ITEM_LOADED_FROM = 'MANUAL'  -- should this be removed?
     AND SCHEDULED_SURFACE_ID = 'NEW_TAB_EN_US'
     AND NOT is_syndicated
     AND NOT is_collection
-    
-    )
+),
+
+all_recent_items as (
```

Comment:
I'd probably combine these two CTEs (which has no functional difference), curious if G sees any performance difference
```
deduped as (
  SELECT * FROM (
    SELECT * FROM recently_scheduled_items
    UNION ALL 
    SELECT * FROM recently_updated_items )
  QUALIFY row_number() OVER (PARTITION BY ID ORDER BY REVIEW_TIME DESC) = 1
)
```

Reply:
There is no performance difference. the `all_recent_items` is just an alias for the same UNION ALL sql you are doing. Mathijs may have done it for readability reasons